docs/machine-learning/how-to-guides/prepare-data-ml-net.md

```diff
@@ -0,0 +1,309 @@
+---
+title: "How-To: Prepare Data with ML.NET"
+description: Learn how to use transforms in ML.NET to manipulate and prepare data for additional processing or model building.
+author: luisquintanilla
+ms.author: luquinta
+ms.date: 04/09/2019
+ms.custom: mvc,how-to
+#Customer intent: As a developer, I want to know how I can transform and prepare data with ML.NET using transforms
+---
+
+# How-To: Prepare Data with ML.NET
+
+This how-to shows how to use transformers in ML.NET to prepare data for additional processing or building a model.
```

Comment:
I think this would be a good place to describe features in general, and the requirements ML.NET places on the Features column.